helper/src/commonTest/kotlin/relatedItems/TestSearcherConnectionRelatedItems.kt

```diff
@@ -0,0 +1,59 @@
+package relatedItems
+
+import com.algolia.instantsearch.core.hits.HitsView
+import com.algolia.instantsearch.helper.relateditems.MatchingPattern
+import com.algolia.instantsearch.helper.relateditems.connectRelatedHitsView
+import com.algolia.instantsearch.helper.searcher.SearcherSingleIndex
+import com.algolia.search.helper.deserialize
+import com.algolia.search.model.Attribute
+import com.algolia.search.model.IndexName
+import com.algolia.search.model.ObjectID
+import mockClient
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class TestSearcherConnectionRelatedItems {
+
+    private val client = mockClient()
+    private val index = client.initIndex(IndexName("Index"))
+
+    @Test
+    fun connect() {
+        val searcher = SearcherSingleIndex(index)
+        val adapter = mockHitsView<Product>()
+
+        val patternBrand = MatchingPattern(Attribute("attBrand"), 1, Product::brand)
+        val patternCategories = MatchingPattern(Attribute("attCategories"), 1, Product::categories)
```

Comment:
nitpick: change the score to something different than 1 just to have it different than the one above